tests/phpunit/CRM/Contribute/Form/ContributionTest.php

```diff
@@ -374,7 +374,7 @@ public function testSubmitCreditCardNoReceipt() {
    */
   public function testSubmitCreditCardFee() {
     $form = new CRM_Contribute_Form_Contribution();
-    $this->paymentProcessor->setDoDirectPaymentResult(['is_error' => 0, 'trxn_id' => 'tx', 'fee_amount' => .08]);
+    $this->paymentProcessor->setDoDirectPaymentResult(['payment_status_id' => 1, 'is_error' => 0, 'trxn_id' => 'tx', 'fee_amount' => .08]);
```

Comment:
OK - I see this - setDoDirectPaymentResult is a test only function & it is now required that payment_status_id is set if it is used - which seems OK

Reply:
Yes. setting that property basically overrides the entire processing within `doPayment()` so we're no longer testing the payment processor but the form submission and completion.